ui/elements/Carousel/Carousel.tsx

```diff
@@ -34,53 +34,74 @@ const Carousel = (props: CarouselProps) => {
     const headerRef = useRef<HTMLDivElement | null>(null);
     const containerRef = useRef<HTMLDivElement | null>(null);
 
-    const [translateX, setTranslateX] = useState(0);
+    // const [translateX, setTranslateX] = useState(0);
```

Comment:
lets remove this if this is not required anymore instead of turning this into a comment